README.md

```diff
@@ -3,30 +3,41 @@
 [![Build Status][build-badge]][build-link]
 [![Apache 2.0 License][license-badge]][license-link]
 
-The SDK has dependencies on specific versions of the C++ WebAssembly toolchain Emscripten (https://emscripten.org) and the protobuf library, therefor use of a Docker image is recommended.
+The SDK uses [bazel](https://bazel.build/) for its build. 
 
-## Docker
 
-A Dockerfile for the C++ SDK is provided in Dockerfile-sdk.
+## Creating a project for use
 
-It can built in this directory by:
+Create a directory with your WORKSPACE file, source files, and BUILD files:
+
+WORKSPACE file (WORKSPACE):
 
 ```
-docker build -t wasmsdk:v2 -f Dockerfile-sdk .
-```
+workspace(name = "my_extension")
 
-The docker image can be used for compiling wasm files.
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
-### Creating a project for use with the Docker build image
+# Point to local proxy wasm sdk. 
+# If you want to import remote proxy wasm sdk, use the `http_archive` rule below.
+local_repository(
+    name = "proxy_wasm_cpp_sdk",
+    path = "<path-to-local-proxy-cpp-sdk>",
+)
```

Comment:
Majority of users won't be using custom C++ SDK, so the instruction should use `http_archive` by default.

Reply:
Cleaned up.